app/src/main/res/values/styles.xml

```diff
@@ -6,6 +6,10 @@
         <item name="android:indeterminateTint">@color/black</item>
     </style>
 
+    <style name="FragmentLoadingBarStyle">
```

Comment:
as you here are adding only one property and you use this style only in one place, you can remove it and put the property on the view

Reply:
Done thx